packages/flutter/lib/src/rendering/view.dart

```diff
@@ -273,22 +273,27 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
       // height is kept as top window padding.
       _window.padding.top / 2.0,
     );
-    // Center of the navigation bar
-    final Offset bottom = Offset(
-      // Horizontal center of the screen
-      bounds.center.dx,
-      // Vertical center of the system navigation bar. The system navigation bar
-      // height is kept as bottom window padding. The "1" needs to be subtracted
-      // from the bottom because available pixels are in (0..bottom) range.
-      // I.e. for a device with 1920 height, bound.bottom is 1920, but the most
-      // bottom drawn pixel is at 1919 position.
-      bounds.bottom - 1.0 - _window.padding.bottom / 2.0,
-    );
     final SystemUiOverlayStyle? upperOverlayStyle = layer!.find<SystemUiOverlayStyle>(top);
```

Comment:
Would the same logic apply to the system status bar? Can this be skipped if there isn't viewPadding at the top?

Reply:
Good idea, let me check that...

Reply:
can you hide this on iOS or just Android?

Reply:
I'll follow up with this one another time